plone-5/instance/src/ukstats.csv_type/src/ukstats/csv_type/profiles/default/types/csv_type.xml

```diff
@@ -29,9 +22,9 @@
   <!-- Schema, class and security -->
   <property name="add_permission">ukstats.csv_type.AddCSVType</property>
   <property name="klass">plone.dexterity.content.Container</property>
-  <property name="model_file">ukstats.csv_type.content:csv_type.xml</property>
```

Comment:
the XML file is still in the folder - is that expected?

Reply:
I've deleted the model xml